brainscore_language/__init__.py

```diff
@@ -69,4 +107,18 @@ def score(model_identifier: str, benchmark_identifier: str) -> Score:
     score: Score = benchmark(model)
     score.attrs['model_identifier'] = model_identifier
     score.attrs['benchmark_identifier'] = benchmark_identifier
+    save_score(score)
+    
     return score
+
+
+def get_score(model_identifier: str, benchmark_identifier: str, create_env=False) -> Score:
```

Comment:
I think this function is `score`, and the current `score` we make into a private `_run_score` method.